e2e/testdata/makebucket.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: make-bucket
+  namespace: default
+spec:
+  template:
+    spec:
+      restartPolicy: OnFailure
+      containers:
+        - command:
+            - s3cmd
+            - --host=minio.default.svc:9000
+            - --host-bucket=minio.default.svc:9000
+            - --access_key=minioadmin
+            - --secret_key=minioadmin
+            - --ssl
+            - --no-check-certificate
```

Comment:
This seems unnecessary.

Reply:
fixed at https://github.com/cybozu-go/moco/pull/580/commits/e9dfe5430ce1a74560f94bf8aa63ce3f8c607ccf